### PR TITLE
Rg palindromo exercise

### PR DESCRIPTION
**Pull request by RodoGalofre**

### CODE REVIEW

File: Palindromo.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+puts "Palindromo"
+
+class PalindromoExcersise
```
Comment (march23) [inline Palindromo.rb]: is this a name for a class ? 
is this a name for a method?
what do you mean with piec?

let's review how a class should be defined and why

avoid mixin languages

File: Palindromo.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+def calcular_palindromo_3_cifras()
+
+    palindromo_mayor = 0
+    for i in 100..999
+        for j in 100..999
+            posible_p = i*j
+            piec = posible_p.to_s
+            if piec == posible_p.to_s.reverse 
+                   
+                #puts " #{piec} es un palindromo"
```
Comment (march23) [inline Palindromo.rb]: avoid push commented lines on your code

File: Palindromo.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+            # else puts " no es un palindromo "
+            end
+        end
+       #puts " "
+    end
+    puts "Palindromo mayor = #{palindromo_mayor}"
+end
+
+obj = PalindromoExcersise.new
```
Comment (march23) [inline Palindromo.rb]: how this works ?
which params are you receiving?

let's review a class definition

**Review by march23 — COMMENTED** [review]
- this PR's does not have a description
- avoid mixin languages
- avoid unused code
- review a class definition

File: Palindromo.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
+def calcular_palindromo_3_cifras(limit)
+  last_palindrome = 0
+  for i in 100..limit
+    for j in 100..limit
+      posible_p = i*j
+      palindrome_valid = posible_p.to_s
+      if palindrome_valid == posible_p.to_s.reverse 
+        if palindrome_valid.to_i > last_palindrome.to_i
```
Comment (danielosorio83) [inline Palindromo.rb]: podrias colocar esta condicion con un && en la condicion del if arriba y tiene el mismo efecto. no hay necesidad de colocar 2 if anidados si no hay mas nada

**Review by danielosorio83 — CHANGES_REQUESTED** [review]

**Review by danielosorio83 — APPROVED** [review]

**Pull request merged**